matrix_client/api.py

```diff
@@ -378,7 +378,7 @@ def media_upload(self, content, content_type):
     def get_display_name(self, user_id):
         content = self._send("GET", "/profile/%s/displayname" % user_id)
         if "displayname" not in content.keys():
-            raise MatrixUnexpectedResponse("'displayname' missing")
+            return user_id
         return content['displayname']
```

Comment:
Probably could go for ``return content.get('avatar_url', None)`` to save a few lines. I'd also prefer None over empty string since *technically* they are different.